plugins/mac-videotoolbox/encoder.c

```diff
@@ -18,6 +18,12 @@
 	     codec_type_to_print_fmt(codec_type), ##__VA_ARGS__)
 #define VT_BLOG(level, format, ...) VT_LOG_ENCODER(enc->encoder, enc->codec_type, level, format, ##__VA_ARGS__)
 
+enum aq_mode {
+	AQ_AUTO = -1,
+	AQ_DISABLED,
+	AQ_ENABLED,
+};
```

Comment:
I'd prefer having an "invalid" value start at 0, followed by actual "correct" values, as that way comparison against `0` is always an error (e.g. when comparing against a zero-initialised values). I've seen too many issues that arise because zero was "valid" and code ended up in undefined state.

```suggestion
enum aq_mode {
    AQ_INVALID = 0
	AQ_AUTO,
	AQ_DISABLED,
	AQ_ENABLED,
};
```